megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -605,6 +619,7 @@ private static StringBuilder inGameValues(Entity entity, Player localPlayer) {
             // "Has not yet moved" only during movement phase
             if (!entity.isDone() && game.getPhase() == GamePhase.MOVEMENT) {
                 result.append(addToTT("NotYetMoved", BR));
+                result.append("<BR>");

```

Comment:
In "strictly compliant" html, <br> tags are self-terminating. Doesn't really matter here, but it's the principle of the thing.

Reply:
<br/>

Reply:
Ok, let's keep it as <BR> then.